src/scripts/dungeons/Dungeon.ts

```diff
@@ -691,7 +691,7 @@ dungeonList['Reversal Mountain'] = new Dungeon('Reversal Mountain',
     [
         new DungeonBossPokemon('Cacturne', 24000000, 100),
         new DungeonBossPokemon('Excadrill', 26000000, 100),
-        new DungeonBossPokemon('Heatran', 8000000, 100, new GymBadgeRequirement(BadgeEnums.Elite_UnovaChampion)),
+        new DungeonBossPokemon('Heatran', 50000000, 100, new GymBadgeRequirement(BadgeEnums.Elite_UnovaChampion)),
```

Comment:
Is this supposed to be almost double the other bosses HP?

Reply:
well these 2 bosses are only available after beating iris. i was gonna go with around 32-35 mil at first, so we can do that